src/Npgsql/NpgsqlReadBuffer.cs

```diff
@@ -358,27 +361,51 @@ public ReadOnlyMemory<byte> ReadMemory(int len)
 
         #region Read Complex
 
-        public ValueTask<int> ReadBytes(byte[] output, int outputOffset, int len, bool async)
+        public int Read(Span<byte> output)
+        {
+            var readFromBuffer = Math.Min(ReadBytesLeft, output.Length);
+            if (readFromBuffer > 0)
+            {
+                new Span<byte>(Buffer, ReadPosition, readFromBuffer).CopyTo(output);
+                ReadPosition += output.Length;
```

Comment:
This looks like a (non-critical) bug (which was there before) - it should be readFromBuffer. Assuming I'm not missing something, can you please submit a patch for this against 4.1?

Reply:
Fixed.
Do you expect a PR targeting the `hotfix/4.1.4` branch?

Reply:
I created PR #2970